quickjs.c

```diff
@@ -7048,7 +7048,7 @@ static int JS_PRINTF_FORMAT_ATTR(3, 4) JS_ThrowTypeErrorOrFalse(JSContext *ctx,
     }
 }
 
-#ifdef __GNUC__
+#if defined(__GNUC__) || defined(__clang__)
```

Comment:
I thought clang also defined GNUC, doesn't it?

Reply:
At least locally that seems to not be the case when compiling on Windows, though not sure why